src/modules/sharepoint/list-item-filter/and.list-item-filter.ts

```diff
@@ -0,0 +1,9 @@
+import { ListItemFilter } from './list-item-filter.interface';
+
+export class AndListItemFilter {
+  constructor(private readonly firstFilter: ListItemFilter, private readonly secondFilter: ListItemFilter) {}
```

Comment:
This is great -- I feel if this was going to be a larger code base, I'd be tempted to suggest using `.join` and taking an array of items to join, but its really not needed here